sympy/series/series.py

```diff
@@ -6,7 +6,56 @@
 def series(expr, x=None, x0=0, n=6, dir="+"):
     """Series expansion of expr around point `x = x0`.
 
-    See the docstring of Expr.series() for complete details of this wrapper.
+       Parameters
```

Comment:
It is better to dedent all the following lines by 3-spaces.
We generally expect every things to be indented 4*n characters

Reply:
I am sorry. I couldn't understand you.

Reply:
He meant to say, you have to move all your changes back 3 spaces.
The text should be in line with the first `"`

```python
"""This is the heading

Rest of the test should be indented like this
   Not like this 
```

Reply:
Alright. fine. Thanks.

Reply:
What about 4*n characters?

Reply:
The python code conventions state that every indent should be exactly 4 characters
The Parameters in your code starts at `4+4+3 = 11` characters which violates the conventions.
So, it is better to start the text at same indent as `"` which is `4+4` characters here

Reply:
ok. thanks.